v2/bundleworkload/workload.go

```diff
@@ -0,0 +1,41 @@
+package bundleworkload
+
+import (
+	"context"
+	"errors"
+
+	"github.com/spiffe/go-spiffe/v2/spiffebundle"
+	"github.com/spiffe/go-spiffe/v2/spiffeid"
+)
+
+var ErrNotReady = errors.New("bundle-workload: not ready")
```

Comment:
I'll leave that up to future refactors. The important bit here is that there is a specific error that can be returned to indicate that the operation failed because the workload had yet to receive responses from the Workload API.